package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-iiot-opcua",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
These are the versions I've been using internally. The previous version released by the original developer was 3.12, so I'm planning to bump the version to 4.0 as part of the NPM release Issue (since NPM uses the package.json version field as their release version).